_includes/doc/admin-guide/options/cloud-auth.md

```diff
@@ -0,0 +1,15 @@
+## cloud-auth()
+
+*Description:* Configures cloud-based authentication for the syslog-ng-otlp() destination. This option enables OAuth2 authentication for gRPC connections using the syslog-ng cloud authentication framework.
+
+The following authentication methods are available:
+
+### oauth2()
+
+Configures OAuth2 authentication for gRPC-based destinations. Tokens are automatically injected into gRPC metadata for each request.
```

Comment:
it works in http destinations as well
in that case its content added to the HTTP header